inst/templates/ad_adcm.R

```diff
@@ -34,7 +38,7 @@ adcm <- cm %>%
     date_imputation = "first",
     time_imputation = "first",
     min_dates = list(TRTSDT)
-  ) %>%
+    ) %>%
```

Comment:
The trailing parenthesis should always be aligned with the function name, so

```
derive_vars_dtm(
  date_imputation = "first",
  time_imputation = "first",
  min_dates = list(TRTSDT)
) %>%
```

rather than

```
derive_vars_dtm(
  date_imputation = "first",
  time_imputation = "first",
  min_dates = list(TRTSDT)
  ) %>%
```

Reply:
Done.